zhaquirks/niko/switch.py

```diff
@@ -0,0 +1,300 @@
+"""Niko switches (only Niko Double connectable switch,10A (552-721X2) at this time)"""
+
+from zigpy import types as t
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+from zigpy.zcl.clusters.general import (
+    Basic,
+    GreenPowerProxy,
+    Groups,
+    Identify,
+    OnOff,
+    Ota,
+    Scenes,
+)
+from zigpy.zcl.clusters.homeautomation import Diagnostic
+from zigpy.zcl.clusters.manufacturer_specific import ManufacturerSpecificCluster
+
+from zhaquirks import EventableCluster
+from zhaquirks.const import (
+    ARGS,
+    BUTTON_1,
+    BUTTON_2,
+    BUTTON_3,
+    BUTTON_4,
+    COMMAND_CLICK,
+    COMMAND_HOLD,
+    COMMAND_RELEASE,
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+from zhaquirks.niko import NIKO
+
+
+class NikoConfigCluster(CustomCluster, ManufacturerSpecificCluster):
```

Comment:
Remove `ManufacturerSpecificCluster` here. AFAIK it's only really used internally by zigpy.